lib/gui/app/modules/analytics.js

```diff
@@ -23,8 +23,10 @@ const settings = require('../models/settings')
 
 resinCorvus.install({
   services: {
-    sentry: _.get(packageJSON, [ 'analytics', 'sentry', 'token' ]),
-    mixpanel: _.get(packageJSON, [ 'analytics', 'mixpanel', 'token' ])
+    sentry: process.env.ANALYTICS_SENTRY_TOKEN ||
+      _.get(packageJSON, [ 'analytics', 'sentry', 'token' ]),
+    mixpanel: process.env.ANALYTICS_MIXPANEL_TOKEN ||
```

Comment:
I think we should prefix these with `ETCHER_`, but that'd deviate from the build process, wdyt @resin-io/etcher?